R/install.R

```diff
@@ -312,9 +315,14 @@ install_torch <- function(version = "1.7.1", type = install_type(version = versi
     dir.create(path, showWarnings = FALSE)
   }
   
+  saved_timeout <- getOption('timeout')
+  options(timeout = timeout)
```

Comment:
Could we use `withr::with_options(list(timeout = ...)` instead? We might not get into the `options(timeout = saved_timeout)` call if the function errors in between.

Reply:
Sure ! I'll do it, thanks for the review .